src/foam/nanos/auth/Group.js

```diff
@@ -131,6 +131,19 @@ foam.CLASS({
       documentation: 'Custom authentication settings for this group.'
     }
     */
+    {
+      class: 'Boolean',
+      name: 'flag',
+      documentation: 'flag to filter sme specific groups'
```

Comment:
Let's rename this property to **template** and change the documentation to _Used to specify that the group is used as a reference for the creation of other groups._